packages/shared/dao/sagas.ts

```diff
@@ -165,10 +167,10 @@ function* selectRealm() {
     (PREVIEW ? PREVIEW_REALM : null) ||
     // CATALYST from url parameter
     (PIN_CATALYST ? realmFromPinnedCatalyst() : null) ||
-    // cached in local storage
-    (yield call(getRealmFromLocalStorage, network)) ||
     // fetch catalysts and select one using the load balancing
-    (yield call(pickCatalystRealm))
+    (yield call(pickCatalystRealm)) ||
+    // cached in local storage
+    (yield call(getRealmFromLocalStorage, network))
```

Comment:
does it makes sense to have this at all? if pickCatalystRealm was unable to pick one won't that mean that the servers are down or something?